### ISSUE TITLE
Switch fork dependencies (timex, timex_ecto, extwitter) over to official packages

### ISSUE THREAD

**Issue by PragTob** [rrrene/elixirstatus-web#37]
Hi there,

thanks for this project and all the other projects you do in Elixir land :)

I'd suggest switching extwitter & friends over to the official package again. It gave me this nice warning when installing: `warning: the dependency :extwitter requires Elixir "~> 1.0.0" but you are running on v1.2.5` - it's also ~140 commits behind their master.

I see you have commits on the fork, if they are still needed we should work on getting those in there and then switching it over. If for some reason your changes are not accepted over there, then I'd suggest putting them into an alone standing package so that they can work with the main package (or even just put the code into this project if that's the only place they are used).

Happy to help you with this endeavour :)

**Comment by rrrene** [comment]
Yeah, I figured as much. Especially when compiled with 1.3, this project needs some updating.

I will see to it after the Credo v0.4 release and let you know if you can help!
